index.js

```diff
@@ -1,7 +1,7 @@
 module.exports = function(el, options) {
   options = options || {};
 
-  var elementDocument = el.ownerDocument;
+  var elementDocument = el.ownerDocument || el;
```

Comment:
Are we certain that `document` is the only Node that does not have `ownerDocument`? Or should we check `el instanceof Document ? el : el.ownerDocument`?

Reply:
From [W3C DOM4](https://www.w3.org/TR/dom/#dom-node-ownerdocument):

> Each **node** has an associated **node document**, set upon creation, that is a **document**. 
> [...]
> The **ownerDocument** attribute must run these steps:
> 1. If the **context object** is a **document**, return null.
> 2. Return the **node document**. 

[MDN](https://developer.mozilla.org/en-US/docs/Web/API/Node/ownerDocument) indicates that all browsers are compatible with this API

Reply:
Awesome, thanks @felipeochoa.